File: laba 1 sem/laba 4/laba 4 pas/4.2/5.pas
function lenght(str: string): integer;
  var 
    i: integer;
    s: string;
  begin    
    i := 1;
    while s <> str do
      begin
      s += str[i];
      i += 1
    end;
  Result := i - 1;
end;
var
  word: string;

begin
  Write('Введите слово: ');
  ReadLn(word);

  if length(word) >= 2 then
    WriteLn(Copy(word, 1, 2));
  if length(word) >= 3 then
    WriteLn(Copy(word, 1, 3));
  if length(word) >= 4 then
    WriteLn(Copy(word, 1, 4));

end.

File: laba 1 sem/laba 6/function/5.py
def lenght(string):
    count = 0
    for i in string:
        count += 1
    return count

a = [1,5,3,1,6,3]

def f(n):
    b = lenght(n)
    count = 0
    for i in (n):
        count +=i
    return b,count

print(f(a))